lingua_franca/time.py

```diff
@@ -80,3 +91,18 @@ def to_local(dt):
         return dt.astimezone(tz)
     else:
         return dt.replace(tzinfo=gettz("UTC")).astimezone(tz)
+
+
+def to_system(dt):
+    """Convert a datetime to the system's local timezone
```

Comment:
this method had not been migrated from mycroft-core, added in order to deprecate the mycroft side function as described in https://github.com/MycroftAI/lingua-franca/issues/182#issuecomment-809776181